x-pack/plugins/apm/public/store/urlParams.ts

```diff
@@ -102,6 +102,15 @@ function toString(str?: string | string[]) {
   return str;
 }
 
+export function toBoolean(value?: string | boolean) {
+  if (value === 'true' || value === true) {
+    return true;
+  }
+  if (value === 'false' || value === false) {
+    return false;
+  }
+}
```

Comment:
I think these conversion methods should only expect the raw url param, which is always a string. In that case this is sufficient:

```js
export function toBoolean(value?: string) {
  return value === 'true';
}
```